src/gwt/src/org/rstudio/studio/client/common/dependencies/DependencyManager.java

```diff
@@ -421,8 +426,9 @@ public void withReticulate(final String progressCaption,
             progressCaption,
             userPrompt,
             new Dependency[] {
+                  Dependency.cranPackage("jsonlite", "0.9.19"),
+                  Dependency.cranPackage("png", "0.1-7"),
                   Dependency.cranPackage("reticulate", "1.6"),
```

Comment:
We might also want to bump the reticulate dependency to 1.10 (the latest version on CRAN).

Reply:
Good idea! Done.